var/spack/repos/builtin/packages/gaussian-view/package.py

```diff
@@ -13,29 +14,78 @@ class GaussianView(Package):
     Gaussian. It aids in the creation of Gaussian input files, enables the
     user to run Gaussian calculations from a graphical interface without the
     need for using a command line instruction, and helps in the interpretation
-    of Gaussian output"""
+    of Gaussian output.
+    Needs post-install steps to make it run!
```

Comment:
Same concern as in main Gaussian recipe.

Reply:
We should add an empty line before this line so that it won't end up in the module file.

Reply:
done, thanks